app/scripts/modules/assert_package_bumps_standalone.sh

```diff
@@ -32,8 +32,9 @@ HAS_PURE_PKG_BUMP=false
 for PKGJSON in */package.json ; do
   MODULE=$(basename "$(dirname "$PKGJSON")")
 
+  IS_PRIVATE_PKG=$(jq .private $PKGJSON)
   HAS_PKG_BUMP=$(git diff -U0 "$TARGET_BRANCH" -- "$PKGJSON" | grep -c '"version":')
-  if [ "$HAS_PKG_BUMP" -ne 0 ] ; then
+  if [ $IS_PRIVATE_PKG != "true" ] &&  [ "$HAS_PKG_BUMP" -ne 0 ] ; then
```

Comment:
Adding a new fix here to ignore changes in private package json files. Otherwise the build will fail for this PR.